ui/plugins/controller.js

```diff
@@ -339,7 +339,20 @@ treeherder.controller('PluginCtrl', [
                    ($scope.job.state === "pending" || $scope.job.state === "running");
         };
 
-        $scope.retriggerJob = function (jobs) {
+        $scope.spinButton = function (selector, spinclass) {
+            $( selector ).removeClass( spinclass );
+            $( selector ).addClass( spinclass );
+            $timeout(function () {
+                $( selector ).removeClass( spinclass );
+            }, 600);
+        };
+
+        $scope.retriggerJob = function (jobs, evt) {
+            // Pulse the retrigger button when it is clicked to give feedback.
+            if (evt) {
```

Comment:
I think we could skip the evt stuff and just do this unconditionally. It would be a nice hint if the user pressed the r button.